src/legacy/ui/public/vislib/__tests__/lib/fixtures/dispatch_bar_chart_config_normal.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
A lot of the stuff in the fixtures are redundant and we have to touch a lot of files if any of this file structure changes in the future plus it's difficult to grasp the differences in configurations because there is just so much code. Maybe it makes sense to have functions which create these configs which can re-use parts (e.g. a `createDispatchBarChartConfig()` which uses `createBarSeriesParams` or something). But this is just a suggestion, I'm not sure whether this is too much effort for too little value

Reply:
I do agree that that would make sense, if we'd make more tests in this way in vislib, for now I'd leave it this way (Im thinning out data). furthermore, in the future the army of `elastic-charts` will take over here, right?